hmpps-prisoner-search/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/services/attributesearch/api/AttributeSearchRequest.kt

```diff
@@ -7,19 +7,33 @@ import uk.gov.justice.digital.hmpps.prisonersearch.search.services.dto.Paginatio
 
 @Schema(description = "A request to search for prisoners by attributes")
 data class AttributeSearchRequest(
-  @Schema(description = "A query to search for prisoners by attributes")
-  val query: Query,
+  @Schema(description = "The type of join to use when combining the matchers and subQueries", example = "AND", defaultValue = "AND")
+  val joinType: JoinType = JoinType.AND,
+  @Schema(description = "A list of queries of type Query that will be combined with the matchers in this query")
+  val queries: List<Query> = listOf(),
```

Comment:
The main change is to remove the top level `Query` and replace it with a `List<Query>`. This feels more natural to use when a search consists of multiple queries.